ConsentViewController/Classes/SPDeviceManager.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  SPDeviceManager.swift
+//  ConsentViewController
+//
+//  Created by Andre Herculano on 23.12.20.
+//
+
+import Foundation
+
+protocol SPDeviceManager {
+    func osVersion () -> String
+    func deviceFamily () -> String
+}
+
+struct SPDevice: SPDeviceManager {
+    /// Returns he Version of the OS. E.g 1.2
```

Comment:
Its typo instead of `he` we should change to `the

Reply:
thanks @vilasma 👍